templates/zerver/confirm_continue_registration.html

```diff
@@ -0,0 +1,46 @@
+{% extends "zerver/portico_signup.html" %}
+
+{% block portico_content %}
+<div class="app register-page">
+    <div class="app-main register-page-container">
+        <div class="center-container">
+            <div class="center-block">
+                <div class="register-form">
+                    <p class="lead">
+                        <div class="register-page-header">{{ _("Account not found!") }}…</div>
+                    </p>
+                    {% if invalid_email %}
+                    <p>
+                        {% trans %}
+                        Please click the following button if you wish to register.
+                        {% endtrans %}
+                    </p>
```

Comment:
Right, which means somehow their OAuth provider sent an invalid-format email, right?  I feel like we may just not need this condition if so.  Or am I missing something?